pkg/users/account-create-dialog.js

```diff
@@ -123,6 +132,26 @@ function validate_real_name(real_name) {
         return _("The full name must not contain colons.");
 }
 
+function validate_uid(uid, accounts, min_id, max_id, change) {
+    if (!uid)
+        return undefined;
+
+    const uid_number = Number(uid);
+    if (!Number.isInteger(uid_number) || uid_number < 0)
+        return _("User ID must be a positive integer");
+
+    if (uid_number < min_id)
+        return cockpit.format(_("User ID must not be lower than $0"), min_id);
+
+    if (uid_number > max_id)
+        return cockpit.format(_("User ID must not be higher than $0"), max_id);
```

Comment:
[This is an interesting failure](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18353-20230301-144211-571cac6c-ubuntu-2204/log.html#334-2). The reason for the error may be a flake, but the error message is broken: "User ID must not be higher than", so apparently `max_id` is `null` here. It's very likely that `min_id` is `null` as well. Note: `3 > undefined` and `3 < undefined` are both false, while `3 > null` is true and `3 < null` is false, i.e. `null` behaves like `0` while comparing against `undefined` is always false. Go, JS!